snap/info.go

```diff
@@ -36,7 +36,10 @@ type PlaceInfo interface {
 	// Name returns the name of the snap.
 	Name() string
 
-	// MountDir returns the base directory of the snap.
+	// BaseDir returns the base directory for all versions of the snap.
+	BaseDir() string
```

Comment:
Let's please drop all BaseDir entries. The use case in the template is better handled by exposing App to the template directly, and using the proper method names instead of rebuilding the paths inside the template.